### PR TITLE
Modify DA Test to use SubmitBlocks with multiple blocks

### PR DESCRIPTION
**Pull request by Manav-Aggarwal**
## Overview
Closes: #1086 

## Checklist

- [x] New and updated code has appropriate documentation
- [x] New and updated code has new and/or updated testing
- [x] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [x] Linked issues closed with keywords

### CODE REVIEW

**Review by gupadhyaya — APPROVED** [review]

File: da/test/da_test.go [hunk 1]
```diff
@@ -184,7 +189,7 @@ func doTestRetrieve(t *testing.T, dalc da.DataAvailabilityLayerClient) {
 		assert.Len(ret.Blocks, cnt, h)
```
Comment (nashqueue) [inline da/test/da_test.go:189]: We should add an assertion that countAtHeight is always equal to 10. And maybe parameterize 10 as the number of blocks per height.

**Review by tuxcanfly — APPROVED** [review]
OK.

File: da/test/da_test.go (hunk 1 above)
Comment (nashqueue) [reply da/test/da_test.go:189]: How is it not necessarily ten? Are we not always submitting ten blocks?

Comment (Manav-Aggarwal) [reply da/test/da_test.go:189]: Yes, we are always submitting 10 blocks at a time but if you look closely, we are keeping track of the number of blocks at each DA height not how many blocks were submitted together. Each time we have a `time.Sleep(time.Duration(rand.Int63() % mockDaBlockTime.Milliseconds()))` which means we sleep for a random duration smaller than the `mockDABlockTime`.  If `resp.DA height` remains the same, for example, all the 100 blocks are submitted at one DA height.  So we cannot assert `countAtHeight` is always equal to 10.

Comment (nashqueue) [reply da/test/da_test.go:192]: I see ty for explaining. The closest explicit assert that we can make is to check if countAtHeight is modulo 10 as one batch is always 10 so one height is a multiple of 10 which might be 0.

**Review by nashqueue — APPROVED** [review]
LGTM

**Review by gupadhyaya — APPROVED** [review]

**Pull request merged**